dataland-frontend/src/components/resources/frameworkDataSearch/FrameworkDataSearchFilters.vue

```diff
@@ -0,0 +1,197 @@
+  setup() {
+    return {
+      getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
+    };
+  },
+  props: {
+    selectedFrameworks: {
+      type: Array as () => Array<DataTypeEnum>,
+      default: () => [],
+    },
+    selectedSectors: {
+      type: Array as () => Array<string>,
+      default: () => [],
+    },
+    selectedCountryCodes: {
+      type: Array as () => Array<string>,
+      default: () => [],
+    },
+    showHeading: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  data() {
+    return {
+      route: useRoute(),
+      availableCountries: [] as Array<CountryCodeSelectableItem>,
+      availableFrameworks: [] as Array<FrameworkSelectableItem>,
+      availableSectors: [] as Array<SelectableItem>,
+    };
+  },
+  computed: {
+    selectedCountriesInt: {
+      get(): Array<CountryCodeSelectableItem> {
+        return this.availableCountries.filter((it) => this.selectedCountryCodes.includes(it.countryCode));
+      },
+      set(newValue: Array<CountryCodeSelectableItem>) {
+        this.$emit(
+          "update:selectedCountryCodes",
+          newValue.map((it) => it.countryCode)
+        );
+      },
+    },
+    selectedFrameworksInt: {
+      get(): Array<FrameworkSelectableItem> {
+        return this.availableFrameworks.filter((it) => this.selectedFrameworks.includes(it.frameworkDataType));
+      },
+      set(newValue: Array<FrameworkSelectableItem>) {
+        this.$emit(
+          "update:selectedFrameworks",
+          newValue.map((it) => it.frameworkDataType)
+        );
+      },
+    },
+    selectedSectorsInt: {
+      get(): Array<SelectableItem> {
+        return this.availableSectors.filter((it) => this.selectedSectors.includes(it.displayName));
+      },
+      set(newValue: Array<SelectableItem>) {
+        this.$emit(
+          "update:selectedSectors",
+          newValue.map((it) => it.displayName)
+        );
+      },
+    },
+  },
+  methods: {
+    resetFilters() {
+      this.selectedFrameworksInt = this.availableFrameworks.filter((it) => !it.disabled);
+      this.selectedCountriesInt = [];
+      this.selectedSectorsInt = [];
+    },
+    async retrieveCountryAndSectorFilterOptions() {
+      const companyDataControllerApi = await new ApiClientProvider(
+        this.getKeycloakPromise!!()
+      ).getCompanyDataControllerApi();
+      const availableSearchFilters = await companyDataControllerApi.getAvailableCompanySearchFilters();
+      this.availableCountries = [...availableSearchFilters.data.countryCodes!!].map((it) => {
+        return {
+          countryCode: it,
+          displayName: getCountryNameFromCountryCode(it),
+          disabled: false,
+        };
+      });
+      this.availableSectors = [...availableSearchFilters.data.sectors!!].map((it) => {
+        return { displayName: it, disabled: false };
+      });
+    },
+    retrieveAvailableFrameworks() {
```

Comment:
Available frameworks are independant of the data base.
In my opinion, it would also make sende to only display those frameworks as "available", which have at least one data set.
=> Point of discussion.